models/Asset/Service.php

```diff
@@ -493,4 +499,142 @@ public static function getUniqueKey(ElementInterface $element, int $nr = 0): str
 
         return $key;
     }
+
+    /**
+     * @throws \Exception
+     */
+    public static function getImageThumbnailByArrayConfig(array $assetInfo): null|ImageThumbnail|VideoImageThumbnail|DocumentImageThumbnail|array
```

Comment:
I know I am again a little nitpicky but should we maybe also call the parameter $config then instead of $assetInfo?
```suggestion
    public static function getImageThumbnailByArrayConfig(array $config): null|ImageThumbnail|VideoImageThumbnail|DocumentImageThumbnail|array
```

Reply:
i agree with you, if we are calling it `ByArrayConfig`, it should be `$config`.

But please next time feel free to commit directly after using a replace all or the refactor tool in phpstorm, you almost gave me an "infart" to see all the requested change 😄also it may be time consuming to have to manually suggest the changes on each occurence and then resolving, one line would be enough

Reply:
Yes sorry about this. I was curious about this "Add suggestion to batch" :) 
Of course next time it will be a online. Sorry again!